runtime/src/main/scala/akka/grpc/Trailers.scala

```diff
@@ -4,16 +4,19 @@
 
 package akka.grpc
 
+import akka.annotation.ApiMayChange
 import io.grpc.Status
 import akka.grpc.internal.JavaMetadataImpl
 import akka.grpc.scaladsl.{ Metadata, MetadataBuilder }
 import akka.grpc.javadsl.{ Metadata => jMetadata }
 
+@ApiMayChange
 case class Trailers(status: Status, metadata: Metadata) {
```

Comment:
Could this be turned into a regular class to ease bin-compatible evolvement?

Reply:
possibly, but I'm not yet sure about the `Metadata` API's yet - perhaps we could use more of https://grpc.github.io/grpc-java/javadoc/io/grpc/Metadata.html . Since this seems like a poweruser api it seemed low-cost to make the whole class ApiMayChange for now.